### PR TITLE
Avalonia 11 preview 6 update

### LINKED ISSUE #31: Update to 11.0.0-preview6
I just updated to the new preview of V11 and there's a breaking change for this library:
`Method not found: 'Avalonia.StyledProperty``.

Seems that the lib is using a changed or removed property, just wanted to document it for you!

### PR DESCRIPTION
**Pull request by heuristicAL**
- Updates Avalonia V11 Preview to latest version (6)
- Disable Name generator with CSPROJ field, which fixes the breaking change that makes the XAML name generator run by default.

Fixes #31

### CODE REVIEW

**Comment by heuristicAL** [comment]
@SebastianRumohr Quick PR for your perusal 🙂

**Comment by SebastianRumohr** [comment]
Thank you very much for your contribution.  
One question:
What is `<AvaloniaNameGeneratorIsEnabled>false</AvaloniaNameGeneratorIsEnabled>` for?
Is there any documentation available?

Best regards.

File: demo/Demo.csproj [hunk 1]
```diff
@@ -2,16 +2,17 @@
     <PropertyGroup>
         <OutputType>WinExe</OutputType>
         <TargetFramework>net7.0</TargetFramework>
+		<AvaloniaNameGeneratorIsEnabled>false</AvaloniaNameGeneratorIsEnabled>
```
Comment (SebastianRumohr) [inline demo/Demo.csproj:5]: Could you please reduce the indention to match the rest of the file?

**Review by SebastianRumohr — CHANGES_REQUESTED** [review]

File: demo/Demo.csproj (hunk 1 above)
Comment (heuristicAL) [reply demo/Demo.csproj]: Nice find, my bad!
All done 🙂

**Pull request merged**